lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -136,15 +138,241 @@ void SessionStateEnforcer::handle_session_init_rule_updates(
    */
   auto update_criteria = get_default_update_criteria();
   session_state.set_fsm_state(CREATED, update_criteria);
-  MLOG(MDEBUG) << "State of session changed to "
-               << session_fsm_state_to_str(session_state.get_state());
+  uint32_t cur_version = session_state.get_current_version();
+  session_state.set_current_version(++cur_version, update_criteria);
+  MLOG(MINFO) << "State of session changed to "

```

Comment:
Don't we log this information in `set_fsm_state` already?

Reply:
Removed log  here.